src/styles/global.ts

```diff
@@ -182,6 +183,681 @@ const GlobalStyles = createGlobalStyle`
+    content: '';
+    width: 0.6rem;
+    height: auto;
+    margin: 0 1rem 0 0;
+    display: inline-block;
+    background: #3498db;
+  }
+
+  .Toastify__toast--success {
+    &::before {
+      content: '';
+      width: 0.6rem;
+      height: auto;
+      margin: 0 1rem 0 0;
+      display: inline-block;
+      background: #29a644;
+      order: 1;
+    }
+    &::after {
+      content: '';
+      background: url(${greenCheck}) no-repeat center/contain;
+      display: inline-block;
+      height: 1.8rem;
+      width: 1.8rem;
+      order: 2;
+      margin: auto;
+    }
+  }
+
+  .Toastify__toast--warning::before {
+    content: '';
+    width: 0.6rem;
+    height: auto;
+    margin: 0 1rem 0 0;
+    display: inline-block;
+    background: #f1c40f;
+  }
+
+  .Toastify__toast--error {
+    color: #e74c3c;
+    &::before {
+      content: '';
+      width: 0.6rem;
+      height: auto;
+      margin: 0 1rem 0 0;
+      display: inline-block;
+      background: #e74c3c;
+    }
+  }
+
+  .Toastify__toast-body {
+    margin: auto 0;
+    -ms-flex: 1;
+    flex: 1;
+    padding: 0.8rem 0 0.8rem 0.8rem;
+    order: 3;
+
+    > div {
+      line-height: 1.2;
+    }
+  }
+
+  @media only screen and (max-width: 736px) {
+    .Toastify__toast {
+      margin-bottom: 1rem;
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  .Toastify__close-button {
+    background: transparent;
+    outline: none;
+    border: none;
+    padding: 0;
+    cursor: pointer;
+    opacity: 0.7;
+    transition: 0.3s ease;
+    align-self: flex-start;
+    order: 5;
+    color: #476480;
+    font-weight: var(--font-weight-normal);
+    font-size: 1.4rem;
+    height: 100%;
+    margin: auto 0 auto 1rem;
+
+    &:hover {
+      opacity: 1;
+      background: transparent;
+    }
+  }
+
+  .Toastify__close-button--default {
+    color: #000;
+    opacity: 0.3;
+  }
+
+  .Toastify__close-button:hover,
+  .Toastify__close-button:focus {
+    opacity: 1;
+  }
+
+  @keyframes Toastify__trackProgress {
+    0% {
+      transform: scaleX(1);
+    }
+
+    100% {
+      transform: scaleX(0);
+    }
+  }
+
+  .Toastify__progress-bar {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 0.4rem;
+    z-index: 9999;
+    opacity: 0.7;
+    background-color: rgba(47, 62, 78, 0.2);
+    transform-origin: left;
+  }
+
+  .Toastify__progress-bar--animated {
+    animation: Toastify__trackProgress linear 1 forwards;
+  }
+
+  .Toastify__progress-bar--controlled {
+    transition: transform 0.2s;
+  }
+
+  .Toastify__progress-bar--rtl {
+    right: 0;
+    left: initial;
+    transform-origin: right;
+  }
+
+  .Toastify__progress-bar--default {
+    background: linear-gradient(to right, #4cd964, #5ac8fa, #007aff, #34aadc, #5856d6, #ff2d55);
+  }
+
+  @keyframes Toastify__bounceInRight {
+    from,
+    60%,
+    75%,
+    90%,
+    to {
+      animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
```

Comment:
What's the reason you put `animation-timing-function` inside animation itself, instead of in the class together with `animation-name`?
Is this all just copy-pasta :spaghetti: ? Then ok